### ISSUE TITLE
Add new torch schedulers 

### ISSUE THREAD

**Issue by MattPainter01** [pytorchbearer/torchbearer#623]
Pytorch now has scheduler for cyclic learning rate
